src/iOS/Xamarin.iOS/ViewControllers/SettingsViewController.cs

```diff
@@ -0,0 +1,452 @@
+        {
+            try
+            {
+                _downloadLabel.Text = "Deleting all...";
+                string offlineDataPath = DataManager.GetDataFolder();
+
+                // Delete the entire directory of offline data.
+                Directory.Delete(offlineDataPath, true);
+
+                new UIAlertView(null, "All data deleted", (IUIAlertViewDelegate)null, "OK", null).Show();
+            }
+            catch (Exception exception)
+            {
+                Debug.WriteLine(exception);
+                new UIAlertView("Error", "Couldn't delete the offline data folder", (IUIAlertViewDelegate)null, "OK", null).Show();
+            }
+            finally
+            {
+                _downloadLabel.Text = "Ready";
+            }
+        }
+
+        private void TabChanged(object sender, EventArgs e)
+        {
+            switch (_switcher.SelectedSegment)
+            {
+                case 0:
+                    _aboutView.Hidden = false;
+                    _licensesView.Hidden = _downloadView.Hidden = true;
+                    break;
+
+                case 1:
+                    _licensesView.Hidden = false;
+                    _aboutView.Hidden = _downloadView.Hidden = true;
+                    break;
+
+                case 2:
+                    _downloadView.Hidden = false;
+                    _aboutView.Hidden = _licensesView.Hidden = true;
+                    break;
+            }
+        }
+
+        public override void LoadView()
+        {
+            // Create and configure the views.
+            View = new UIView { BackgroundColor = UIColor.White };
+
+            // Used to switch between the different views.
+            _switcher = new UISegmentedControl(new string[] { "About", "Licenses", "Offline Data" }) { SelectedSegment = 0 };
+
+            // Displays the about.md in a web view.
+            _aboutView = new WKWebView(new CGRect(), new WKWebViewConfiguration());
+            _aboutView.TranslatesAutoresizingMaskIntoConstraints = false;
+            _aboutView.NavigationDelegate = new BrowserLinksNavigationDelegate();
+
+            // Displays the licenses.md in a web view.
+            _licensesView = new WKWebView(new CGRect(), new WKWebViewConfiguration()) { Hidden = true };
+            _licensesView.TranslatesAutoresizingMaskIntoConstraints = false;
+            _licensesView.NavigationDelegate = new BrowserLinksNavigationDelegate();
+
+            // View for managing offline data for samples.
+            _downloadView = new UIStackView { BackgroundColor = UIColor.White, Hidden = true };
+            _downloadView.Axis = UILayoutConstraintAxis.Vertical;
+            _downloadView.LayoutMarginsRelativeArrangement = true;
+            _downloadView.Alignment = UIStackViewAlignment.Fill;
+            _downloadView.TranslatesAutoresizingMaskIntoConstraints = false;
+
+            // Label for download status.
+            _downloadLabel = new UILabel() { Text = "Ready" };
```

Comment:
I recommend centering this, and giving it some vertical margin; maybe `HeightAnchor.ConstraintEqualTo(40)`, but I'll leave specifics to your judgment.